packages/frontend/src/components/admin/ConfirmDeleteModal.tsx

```
@@ -0,0 +1,42 @@
+  open,
+  onOpenChange,
+  onConfirm,
+  entityLabel,
+}: ConfirmDeleteModalProps) {
+  return (
+    <Dialog open={open} onOpenChange={onOpenChange}>
+      <DialogContent>
+        <DialogHeader>
+          <DialogTitle>Confirmer la suppression</DialogTitle>
+        </DialogHeader>
+        <p>Voulez-vous vraiment supprimer cette {entityLabel} ? Cette action est irréversible.</p>
```

ESLint

[error] Irregular whitespace not allowed.

(no-irregular-whitespace)